tower-http/src/trace/on_request.rs

```diff
@@ -60,10 +60,15 @@ impl DefaultOnRequest {
 
     /// Set the [`Level`] used for [tracing events].
     ///
+    /// Please note that while this will set the level for the tracing events
+    /// themselves, it might cause them to lack expected information, like
+    /// request method or path. You can address this using
+    /// [`DefaultMakeSpan::level`].
+    ///
```

Comment:
This bit feels redundant to me. The default span fields are already shown in the example output at the top of [the docs](https://docs.rs/tower-http/0.1.1/tower_http/trace/index.html). I kinda think we should remove this paragraph. Same goes for the change in `tower-http/src/trace/on_response.rs`.

Reply:
Hey @davidpdrsn, I don't understand what you mean here. It's true that the span fields are shown in the example output in the docs. But I was very confused when I used these `level` methods to set the log level for request/response, and then saw log messages that **did not** match this example output. I added the paragraph to address this point of confusion.